src/tikzplotly/_utils.py

```diff
@@ -87,15 +87,20 @@ def sanitize_char(ch: str, keep_space: bool = False) -> str:
     str
         The sanitized character.
     """
+    if ch == "_":
```

Comment:
I think in the context of text (and not TeX text), the character `_` should not be a problem